l10n/en/products/vpn/more/ip-address.ftl

```diff
@@ -21,7 +21,7 @@ vpn-ip-address-does-it-travel = Does it travel with you?
 # Variables
 #   $url (url) - https://www.mozilla.org/products/vpn/more/when-to-use-a-vpn/
 vpn-ip-address-no-your-ip-v2 = No. Your IP address is only associated with one location unless you are using a VPN (we will get more into that later). When you are at your home and connecting to the internet you pay for, you are using one. However, if you check your email at home in the morning, then scan the news at a local coffee shop while waiting for your coffee, and then work from an office, you will have used different IP addresses at each location.
-# Outdated string
+# Obsolete string
```

Comment:
**Nit:** We started adding the expiry dates to obsolete strings so that they'll be easier to clean up. We go with the format of `yyyy-mm-dd`. Here's an example: https://bedrock.readthedocs.io/en/latest/l10n.html#obsolete-strings

Reply:
Right! I'm aware of that and would do it for my own replacements where I know the context for sure — this one is rather old, goes way beyond git blame (deletions & re-additions later etc.), and the rationale was removing a superfluous link I guess, and was communicated via private channels, so I'm not entirely comfortable slapping my own date on that string. (I basically wanted it to just be picked up some time later together with all the other undated obsoletes that may need another look how, when and why they got replaced before really cleaning up.) TL;DR I don't have a proper date for this one to deliberately add (even a +2mo default).

Ah, found it — it was a broken link removed in [#11225 `e1545c4`](https://github.com/mozilla/bedrock/pull/11225/commits/e1545c4f3e593a52c789d108930e89bc3dabf3ed) so I see the rationale now.

I was thinking about just slapping the original obsolescence date to it, like:

```suggestion
# Obsolete string (expires: 2022-04-22)
```

wdyt?

Reply:
Ah, good catch on that one! In that case, feel free to delete it since it's been 2+ years and whatever translations it needed would have already been done by now!